pytype/stubs/builtins/3/builtins.pytd

```diff
@@ -13,6 +13,9 @@ from typing import (
 if sys.version_info >= (3, 8):
     from typing import SupportsIndex
 
+# typing.Annotated, added in Python 3.9
+class Annotated: ...
```

Comment:
This should be in pytype/stubs/builtins/3/*typing*.pytd (not builtins.pytd). I also suggest putting this inside a `sys.version_info >= (3, 9):` block, since the class isn't available prior to 3.9.

Reply:
Gotcha. Done.